api/v1beta1/deployment_types.go

```diff
@@ -68,10 +68,11 @@ type DeploymentStrategy struct {
 // Similar to DeploymentStrategy, it is an annotation used in workload
 // However, unlike DeploymentStrategy, it is only used to store and restore the user's strategy
 type OriginalDeploymentStrategy struct {
-	// The deployment strategy to use to replace existing pods with new ones.
 	// +optional
-	// +patchStrategy=retainKeys
-	Strategy *apps.DeploymentStrategy `json:"strategy,omitempty" patchStrategy:"retainKeys" protobuf:"bytes,4,opt,name=strategy"`
+	MaxUnavailable *intstr.IntOrString `json:"maxUnavailable,omitempty" protobuf:"bytes,1,opt,name=maxUnavailable"`
```

Comment:
OriginalDeploymentStrategy is not part of API, just an internal struct, plz move it out of the api package(maybe to pkg/control?)

Reply:
done